content/messaging/index.mdx

```diff
@@ -65,15 +77,13 @@ The first connection allows for sending an event, for example, a message, by an
 
 Apart from pushes, actions can also trigger the LiveChat's backend to generate webhooks. To receive a webhook, you need to [register it](/management/configuration-api/#register-webhook) first. That's how you let the LiveChat's backend know you want to be notified about certain actions. The LiveChat's backend will send webhooks to the target webhook URL you specify during webhook registration.
 
-## Chat routing
-
-Routing is the process of **assigning chats to agents**. The primary goal of the routing mechanism is to distribute chats to all available agents who use the same license.
+### Flow with integrations
 
-Chats are being assigned to agents **either automatically or manually**. Our default routing mechanisms are described in [Understanding chat routing](https://www.livechatinc.com/help/understanding-chat-routing/).
+Developers can build integrations that extend LiveChat's messaging capabilities, for example, via a new communication channel. Consider the example of the <a href="https://www.livechat.com/marketplace/apps/facebook-messenger/" target="_blank">integration app</a> that connects LiveChat with Facebook Messenger.
```

Comment:
```suggestion
Developers can build integrations that extend LiveChat's messaging capabilities, for example, via a new communication channel. Consider the example of the <a href="https://www.livechat.com/marketplace/apps/facebook-messenger/" target="_blank">integration app</a> that connects LiveChat with Facebook Messenger. Keep in mind the flow presented in the diagram below is typical for a new communication channel and it would be different for an app that simply extends the LiveChat UI.
```